client/src/views/landing.vue

```diff
@@ -22,12 +22,26 @@
                   <sup class="beta-tag">Beta</sup>
                 </v-btn>
               </template>
-              <p>The Name Request website is available as a Beta version. Name Requests obtained through the Beta are
-                official name requests and can be used in the province of British Columbia.</p>
-              <p class="mb-0">As part of a new
-                <a class="white--text" :href="agileUrl" target="_blank">agile software development process</a>,
+              <p>The Name Request website is available as a Beta version. Name requests obtained through the Name
+                Request Beta website are official name requests and can be used in the province of British Columbia.</p>
+              <p class="mb-0">The Name Request Beta website contains all of the functionality of the previous Name
+                Requests Online site, as well as new functionality intended to improve the experience of starting and
+                managing businesses in B.C. Some features of the site may not function as intended, and if you encounter
+                this please contact us:</p>
+              <span class="tooltip-bullet mt-3">
```

Comment:
No need to do this. Use ~components/common/contact-info.vue instead (with `direction="col"`).

Reply:
ah interesting. Will see how this works within a tooltip.

Reply:
FWIW, I think the contact info component, with icons instead of bullets, is fine. (consistency)

Reply:
To use the contact info component here will require changes to the component itself, which will also affect it's other implementation.

Reply:
Oh well. Hopefully we will remove the Beta label at some point :)

Reply:
Definitely. 
That being said, there is similar listed contact info in the lower section as well and an implementation of the contact info component in some grey-boxes etc
This is definitely a spot for some refactoring when we can.